packages/lib/server/repository/routingFormResponse.repository.ts

```
@@ -0,0 +1,33 @@
+        routedToBookingUid: bookingUid,
+      },
+      include: {
+        form: {
+          select: {
+            fields: true,
+          },
+        },
+      },
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)